src/SIL.LCModel/DomainImpl/OverridesLing_Lex.cs

```diff
@@ -1324,7 +1324,9 @@ public ILexEntryRef MakeVariantOf(IVariantComponentLexeme componentLexeme, ILexE
 			}
 			else
 			{
-				ler.VariantEntryTypesRS.Add(Cache.LangProject.LexDbOA.VariantEntryTypesOA.PossibilitiesOS[0] as ILexEntryType);
+				int defVariantIdx = Cache.LangProject.LexDbOA.VariantEntryTypesOA.PossibilitiesOS.Where(e =>
```

Comment:
I was thinking that this could be simplified, but the way it is written has the effect of avoiding any extra conditionals for unit tests where the unspecified variant type might not be in the list.